src/module/rules/index.ts

```diff
@@ -105,7 +105,7 @@ class RuleElements {
             if (REConstructor) {
                 const rule = ((): RuleElementPF2e | null => {
                     try {
-                        return new REConstructor(data, item, options);
```

Comment:
Can this be part of options?

Reply:
Options is passed to the item as a whole, and the source index needs to differ per rule in that item.